app/models/contact.rb

```diff
@@ -21,6 +21,10 @@ class Contact < ApplicationRecord
 
   audited associated_with: :provider
 
+  validates :name, presence: true
+  validates :email, email: true, presence: true
+  validates :telephone, phone: { message: "^Enter a valid telephone number" }, allow_nil: true
```

Comment:
should message be in yml?

Reply:
Same as provider telephone number

Reply:
I think @defong is right, we have a bunch of error messages already in locales, and this one probably belongs there.

https://github.com/DFE-Digital/manage-courses-backend/blob/413be4d4ada4407505ae6371614a929f80fbcf3a/config/locales/en.yml#L67-L68

Reply:
Ah yes, you are right.